client/web/src/features/user/userService.ts

```diff
@@ -0,0 +1,50 @@
+import { IUser } from '../../models/user'
```

Comment:
You don't think we can create a crudService by setting only the type (Serie or User) and the base URL ?
It seems to be same code between userService and seriesService.

Reply:
I don't think because we will have specific methods (for example login method)

Reply:
it is that why I implemented one interface ICrudService